en/docs/WIP/create-and-publish-awslambda-api.md

```diff
@@ -0,0 +1,122 @@
+# Create and Publish an AWS Lambda API
+
+Follow the instructions in this tutorial to design and publish an AWS Lambda API.
+
+<html>
+   <div class="admonition note">
+      <p class="admonition-title">Note</p>
+      <p>For more information on AWS Lambda, see <a href="https://docs.aws.amazon.com/lambda/latest/dg/welcome.html">What is AWS Lambda?</a></p>
+   </div> 
+</html>
+
+### Step 1 - Design a REST API
+
+1. Sign in to the API Publisher Portal.
+   
+    `https://<hostname>:9443/publisher` 
+   
+    Example: `https://localhost:9443/publisher`
+
+    Use your username and password to sign in.
```

Comment:
Enter the username and password to sign in.

Reply:
changed